### PR TITLE
Dolci/paper notebook

### PR DESCRIPTION
**Pull request by Ig-dolci**
The paper was reviewed by David.

### CODE REVIEW

File: docs/paper/paper.md [hunk 1]
```diff
@@ -5,34 +5,106 @@ tags:
...
 # Summary
+The *checkpointing_schedules* provides schedules for step based incremental
+checkpointing of the adjoints to computer models. The schedules contain
+instructions indicating the sequence of forward and adjoint steps to be
+executed, and the data storage and retrieval to be performed. These
+instructions are independent of the model implementation, which enables the
+model authors to switch between checkpointing algorithms without recoding.
+Conversely, *checkpointing_schedules* provides developers of checkpointing
+algorithms a direct mechanism to convey their work to model authors.
+*checkpointing_schedules* has been integrated into **tlm_adjoint** [@Maddison2014],
```
Comment (jrmaddison) [inline docs/paper/paper.md:33]: The correct tlm_adjoint reference is: https://doi.org/10.1137/18M1209465

**Review by jrmaddison — CHANGES_REQUESTED** [review]
See also edits to the tutorial.

**Comment by jrmaddison** [comment]
Also my affiliation is: School of Mathematics and Maxwell Institute for Mathematical Sciences, University of Edinburgh

**Comment by Ig-dolci** [comment]
This PR contains the paper reviewed by David and James. The notebooks tutorial.ipynb was reviewed by James.

**Pull request merged**